### ISSUE TITLE
Pytorch中的转置卷积 | 小王同学

### ISSUE THREAD

**Issue by a171232886** [a171232886/a171232886.github.io#22]
https://a171232886.github.io/2021/11/22/Pytorch%E4%B8%AD%E7%9A%84%E8%BD%AC%E7%BD%AE%E5%8D%B7%E7%A7%AF/ 

 转置卷积(Transposed Convolution)
 又称为转置卷积。
 torch.nn.ConvTranspose2d(in_channels, out_channels, kernel_size, stride=1, padding=0, output_padding=0, groups=1, bias=True, dilation=1, padding_mode='zeros')
